spring-native/src/main/java/org/springframework/aot/test/AotContextLoader.java

```diff
@@ -79,4 +104,8 @@ boolean isSupportedTestClass(Class<?> testClass) {
 		return this.contextLoaders.containsKey(testClass.getName());
 	}
 
+	Class<? extends ApplicationContextInitializer<?>> getContextInitializerClass(Class<?> testClass) {
```

Comment:
I am not against reusing the same component for both purposes (I actually think it's the right call considering the reflection business going on) but I think the class-javadoc should be updated and probably the class renamed as a result.

Reply:
I agree. 

I'll come up with a new name and update the documentation.